lib/bolt/plan_creator.rb

```diff
@@ -85,17 +85,16 @@ def self.create_plan(plans_path, plan_name, outputter, is_puppet)
         run_command  = 'bolt plan run'
       end
 
-      output = <<~OUTPUT
+      # TODO: This should probably return the name and path to the new
```

Comment:
For sure - it doesn't seem like the JSON outputter needs something this readable.